docs/Android/android-install.md

```diff
@@ -120,7 +120,7 @@ def ws1SdkVersion = "24.01"
 
 dependencies {
```

Comment:
is anything needed for the reference to `vmwaresaas.jfrog.io` higher in the file?  (I could not leave a comment on that line.)